### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by b-zee** [libp2p/rust-libp2p#5733] [General]
I'm working at Autonomi and we have a few PRs that were merged recently-ish. (#5507 , #5705) I'm wondering when a next release will be cut. The releases have been quite frequent this year, but the last release was 3 months ago currently. 

Sorry for asking this question. I do not want to put pressure on the rust-libp2p team in any way here. Perhaps there are other channels where discussions surrounding releases happens. Let me know if there's anything I can help with.

**Comment by elenaf9** [comment]
Looking through the list, it would be great to have at least the following PRs included in the next release:
- [x] #5707
- [x] #5715 
- [x] #5727 

@jxs @dariusc93 @guillaumemichel what other PRs / issues should be in before the next relase?

**Reply by dariusc93** [reply]
Maybe https://github.com/libp2p/rust-libp2p/pull/5728 and https://github.com/libp2p/rust-libp2p/pull/5650?

**Reply by jxs** [reply]
yup, thanks Darius, also https://github.com/libp2p/rust-libp2p/pull/4967 and #5728 and I'd like to wait for https://github.com/libp2p/rust-libp2p/pull/5724/ if possible, I know we can then depend on it explicitly after being released, but I'd like to export it via `libp2p`

**Reply by elenaf9** [reply]
I am not sure about #5728; getting the necessary fuzzy testing might take a bit longer, so I wouldn't block on that, see https://github.com/libp2p/rust-libp2p/issues/4650#issuecomment-2537613423. It's nothing urgent anyway, is it?

Waiting for the three other PRs you mentioned:
- [x] #4967
- [x] #5650
- [ ] #5724 

sounds good to me.

**Reply by drHuangMHT** [reply]
> I am not sure about #5728; getting the necessary fuzzy testing might take a bit longer, so I wouldn't block on that, see [#4650 (comment)](https://github.com/libp2p/rust-libp2p/issues/4650#issuecomment-2537613423). It's nothing urgent anyway, is it?

Yeah it will take longer than expected, given the fact that I'm not very familiar with fuzz tests, and `libfuzzer-sys`(therefore `cargo fuzz`) don't seem to work well on my Windows installation so I have to dust off my Manjaro installation.

**Reply by jxs** [reply]
Thanks Elena, updated #5728 with https://github.com/libp2p/rust-libp2p/issues/4650#issuecomment-2545230941. Wrt to #5650 also left https://github.com/libp2p/rust-libp2p/pull/5650#issuecomment-2545207514

**Comment by jxs** [comment]
Hi Benno!
>Sorry for asking this question. I do not want to put pressure on the rust-libp2p team in any way here. Perhaps there are other channels where discussions surrounding releases happens. Let me know if there's anything I can help with.

no pressure at all, this also helps us drive forward with the release process, as when there isn't any formal interest we have been waiting for more changes to accumulate and bundle into a release.
